lua/nvim-treesitter/configs.lua

```diff
@@ -48,6 +48,14 @@ local builtin_modules = {
     enable = false,
     is_supported = queries.has_indents,
   },
+  fold = {
+    module_path = "nvim-treesitter.fold",
+    can_attach = false,
```

Comment:
this can lead to issues when checking, could you reverse the logic ? Something like `do_not_attach` ?